package.json

```diff
@@ -39,13 +39,15 @@
 		"ts-node": "^10.9.1",
 		"tslib": "^2.4.1",
 		"typescript": "^4.9.3",
-		"vite": "^4.0.0",
+		"vite": "^4.4.9",
 		"vitest": "^0.25.3"
 	},
 	"type": "module",
 	"dependencies": {
 		"cross-fetch": "^4.0.0",
 		"crypto-js": "^4.1.1",
-		"js-sha256": "^0.9.0"
+		"js-sha256": "^0.9.0",
+		"svelte-kit": "^1.2.0",
```

Comment:
are you sure this is needed?
https://www.npmjs.com/package/@sveltejs/kit
https://www.npmjs.com/package/svelte-kit (check out this link)

Reply:
You're right